recipes/elot

```diff
@@ -0,0 +1,7 @@
+(elot :fetcher github
+      :repo "johanwk/elot"
+      :files ("elot-package/elot.el"
+              "elot-package/elot-defaults.el"
+              "elot-package/elot-label-display.el"
+              "elot-package/elot-lob.org"
+              "elot-package/theme-readtheorg.setup"))
```

Comment:
While I'm here, I wonder if this would give your package a little more flexibility to grow within its namespace:
```suggestion
      :files ("elot-package/elot*.el"
              "elot-package/elot-lob.org"
              "elot-package/theme-readtheorg.setup"))
```